DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/spatial/spatialIndex/SpatialIndex.java

```diff
@@ -0,0 +1,83 @@
+package net.ravendb.demo.spatial.spatialIndex;
+//region Usings
+import net.ravendb.client.documents.indexes.AbstractIndexCreationTask;
+import net.ravendb.client.documents.session.IDocumentSession;
+//endregion
+import net.ravendb.demo.common.DocumentStoreHolder;
+import net.ravendb.demo.common.models.Company;
+
+import java.util.List;
+
+public class SpatialIndex {
+    //region Demo
+    //region Step_1
+    public static class Companies_ByLocation extends AbstractIndexCreationTask {
+    //endregion
+    
+        //region Step_2
+        public static class IndexEntry {
+            private String companyName;
+            private Object locationCoordinates ;
+        
+            public String getCompanyName() {
+                return companyName;
+            }
+        
+            public void setCompanyName(String companyName) {
+                this.companyName = companyName;
+            }
+        
+            public Object getLocationCoordinates () {
+                return locationCoordinates;
+            }
+        
+            public void setLocationCoordinates (Object locationCoordinates) {
+                this.locationCoordinates = locationCoordinates;
+            }
```

Comment:
please reformat that code and auto generate again

Reply:
done